flytekit/models/node_execution.py

```diff
@@ -185,6 +195,7 @@ def from_flyte_idl(cls, p):
         return cls(
             phase=p.phase,
             output_uri=p.output_uri if p.HasField("output_uri") else None,
+            deck_uri=p.deck_uri if p.deck_uri else None,
```

Comment:
we should check for the existence of the field, otherwise older node executions will not be deserialized.